.about.yml

```diff
@@ -0,0 +1,113 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: project-dashboard
+
+# Full proper name of the project (required)
+full_name: 'Project Dashboard'
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+owner_type: working-group
```

Comment:
Hmm. Even though I said to delete the `parent:` field, it just occurred to me: How to indicate that the `working-group` that owns it is `wg-testing`? Thoughts?